### ISSUE TITLE
Issue loading

### ISSUE THREAD

**Issue by AlexAng-dev** [V1Interactive/FreeformTools#110]
Recently in Maya 2022 in python 2 or 3 there's been an error loading the toolset. I had swapped back and forth python versions and also installed both versions of python.net

From Script Editor

```
Error: No module named maya_startup.user_setup
Traceback (most recent call last):
File "<maya console>", line 1, in <module>
File "C:/Users/Name/Documents/maya/scripts/userSetup.py", line 19, in main
     import maya_startup.user_setup
File "C:\Program Files\Autodesk\Maya2022\Python27\lib\site-packages\shiboken2\files.dir\shibokensupport\__feature__.py", line 142, in _import
     return original_import(name, *args, **kwargs)
ImportError: No module named maya_startup.user_setup # 
Error: No module named maya_startup.user_setup
Traceback (most recent call last):
File "<maya console>", line 1, in <module>
File "C:/Users/Name/Documents/maya/scripts/userSetup.py", line 19, in main
     import maya_startup.user_setup
File "C:\Program Files\Autodesk\Maya2022\Python27\lib\site-packages\shiboken2\files.dir\shibokensupport\__feature__.py", line 142, in _import
     return original_import(name, *args, **kwargs)
ImportError: No module named maya_startup.user_setup # 
```

**Comment by zahmbie** [comment]
Heya, the likely way this error could occur is if the V1TOOLSROOT environment variable isn't set, or isn't set to the correct path.  

From the environment variable path the userSetup.py file looks for this directory path Freeform.Python\DCCUtilities\Maya where there should be a maya_startup folder with user_setup.py inside.  

Could you check if all of that is true on your system?

**Comment by AlexAng-dev** [comment]
Yes my environment variable is set to the folder containing the Freeform.Python sub folder.

**Comment by zahmbie** [comment]
Hmmm, I've got a couple of steps to try if you can get me the output of these code snippets to help debug where it's failing.
  
What does this return?
```
import os
import sys
maya_startup_path = os.path.join(os.environ['V1TOOLSROOT'], 'Freeform.Python', 'DCCUtilities', 'Maya')
print(maya_startup_path in sys.path)
print(os.path.exists(maya_startup_path))
print(os.listdir(maya_startup_path))
```

And once Maya has started after you hit that error what happens when you run this?  
`import maya_startup`

**Comment by AlexAng-dev** [comment]
On running the path script I figured out the environment had a strange double file looking for "FF Rigging" inside of "FF Rigging". After fixing that I got this on startup:


```
# Error: No module named 'Queue'
# Traceback (most recent call last):
#   File "<maya console>", line 1, in <module>
#   File "C:/Users/Name/Documents/maya/scripts/userSetup.py", line 19, in main
#     import maya_startup.user_setup
#   File "C:\Users\Name\Desktop\FF Rigging\Freeform.Python\DCCUtilities\Maya\maya_startup\__init__.py", line 7, in <module>
#     module = loader.find_module(name).load_module(name)
#   File "C:\Users\Name\Desktop\FF Rigging\Freeform.Python\DCCUtilities\Maya\maya_startup\freeform_user_setup.py", line 26, in <module>
#     import Queue
# ModuleNotFoundError: No module named 'Queue' # 
# Error: No module named 'Queue'
# Traceback (most recent call last):
#   File "<maya console>", line 1, in <module>
#   File "C:/Users/Name/Documents/maya/scripts/userSetup.py", line 19, in main
#     import maya_startup.user_setup
#   File "C:\Users\Name\Desktop\FF Rigging\Freeform.Python\DCCUtilities\Maya\maya_startup\__init__.py", line 7, in <module>
#     module = loader.find_module(name).load_module(name)
#   File "C:\Users\Name\Desktop\FF Rigging\Freeform.Python\DCCUtilities\Maya\maya_startup\freeform_user_setup.py", line 26, in <module>
#     import Queue
# ModuleNotFoundError: No module named 'Queue' # 
```


From running the path display script

```
import os
import sys
maya_startup_path = os.path.join(os.environ['V1TOOLSROOT'], 'Freeform.Python', 'DCCUtilities', 'Maya')
print(maya_startup_path in sys.path)
print(os.path.exists(maya_startup_path))
print(os.listdir(maya_startup_path));
// Error: maya_startup_path = os.path.join(os.environ['V1TOOLSROOT'], 'Freeform.Python', 'DCCUtilities', 'Maya')
 // 
// Error: Line 3.19: Syntax error // 
import maya_startup;
// Error: line 1: Cannot find procedure "import". // 

```

After importing

```
import maya_startup
# Error: No module named 'Queue'
# Traceback (most recent call last):
#   File "<maya console>", line 1, in <module>
#   File "C:\Users\Name\Desktop\FF Rigging\Freeform.Python\DCCUtilities\Maya\maya_startup\__init__.py", line 7, in <module>
#     module = loader.find_module(name).load_module(name)
#   File "C:\Users\Name\Desktop\FF Rigging\Freeform.Python\DCCUtilities\Maya\maya_startup\freeform_user_setup.py", line 26, in <module>
#     import Queue
# ModuleNotFoundError: No module named 'Queue' # 
```

**Comment by zahmbie** [comment]
Ahhh, that's progress. :)  Queue is only imported with the Python 2.7 tools and I haven't tested them running Python2 in Maya 2022, but I have worked with the new Python 3 toolset in Maya 2022 quite a bit now.  

I made a new release a couple months ago with some bug fixes and extra functionality like version tracking character files so you know if your animation file has the latest settings loaded.  If you haven't grabbed it already here's the link.  If that still fails for you let me know and we'll continue debugging.

https://github.com/V1Interactive/FreeformTools/releases/tag/37_1.0.1_preview
